src/PresentationalComponents/Notifications/NotificationTemplate.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useFormApi } from '@data-driven-forms/react-form-renderer';
+import PropTypes from 'prop-types';
+import FormButtons from '../shared/FormButtons';
+
+const NotificationsTemplate = ({ schema, formFields }) => {
+  const { handleSubmit } = useFormApi();
+
+  return (
+    <form onSubmit={handleSubmit} style={{ display: 'contents' }}>
+      {schema.title}
```

Comment:
Should we use some sort of element instead of just a plain string?

Reply:
Added `node` to propTypes